mantidproject/mantid
# Elemental Analysis: Peak Data Loading

**Description of work.**
Add selectable Periodic Table that exposes events for: element selection; table changes; table left and right clicks. Loads data from a stock `config.json` that contains peak data - elements (buttons) with no data are disabled.

**Report to:** Anthony Lim

**To test:**
Interfaces -> Muon -> Elemental Analysis
Left clicking an element should turn it yellow, print that the element was left clicked, and add it to a list of elements selected (table changed event). Right clicking an element will log a right click for that element. As an example, the element `Po` should not be clickable.



Fixes #22915 and #22881

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
